specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchindex.json

```
@@ -1855,10 +1855,38 @@
         "fields": {
           "type": "string",
           "description": "Vector Fields of type Collection(Edm.Single) to be included in the vector searched."
+        },
+        "exhaustive": {
+          "type": "boolean",
+          "description": "If set to true, will compute a set of ground truth KNN vectors for evaluating recall."
```

This is too restrictive for the description IMO: *will compute a set of ground truth KNN vectors for evaluating recall.*

Consider this wording instead
```suggestion
          "description": "If set to true, this will perform exhaustive k-nearest neighbors search across all vectors within the vector index of the selected fields."
```